tests/Load/aws-execute-load-tests.sh

```
@@ -0,0 +1,143 @@
+  sleep 5
+done
+
+$AWS_CLI iam add-role-to-instance-profile --instance-profile-name "$ROLE_NAME" --role-name "$ROLE_NAME" --region "$REGION" 2>/dev/null || echo "Role already associated with instance profile. Proceeding..."
+
+echo "Waiting for role to be associated with the instance profile..."
+until $AWS_CLI iam get-instance-profile --instance-profile-name "$ROLE_NAME" --region "$REGION" | grep -q "$ROLE_NAME"; do
+  sleep 5
+done
+
+echo "Checking IAM role permissions..."
+if ! $AWS_CLI $STS_COMMAND; then
```

Shellcheck

[info] Double quote to prevent globbing and word splitting.

(SC2086)